src/Sylius/Bundle/ShopBundle/Resources/views/Order/thankYou.html.twig

```diff
@@ -22,10 +22,13 @@
                     </div>
                 {% endif %}
                 <div class="ui hidden divider"></div>
-                <a href="{{ path('sylius_shop_order_show', {'tokenValue': order.tokenValue}) }}" id="sylius-show-order" class="ui large blue button">{{ 'sylius.ui.view_your_order_or_change_payment_method'|trans }}</a>
+                {% if order.customer.user is not null %}
+                    <a href="{{ path('sylius_shop_account_order_show', {'number': order.number}) }}" id="sylius-show-order-in-account" class="ui large blue button">{{ 'sylius.ui.view_your_order'|trans }}</a>
+                {% endif %}
             {% endif %}
 
             {% if order.customer.user is null %}
+                <a href="{{ path('sylius_shop_order_show', {'tokenValue': order.tokenValue}) }}" id="sylius-show-order" class="ui large blue button">{{ 'sylius.ui.view_your_order_or_change_payment_method'|trans }}</a>
```

Comment:
Shouldn't there be only `sylius.ui.change_payment_method`? As viewing order is not possible for visitors?